server/server_test.go

```diff
@@ -495,7 +495,9 @@ func runTestLoadData(c *C, server *Server) {
 
 func runTestConcurrentUpdate(c *C) {
 	// TODO: Should be runTestsOnNewDB. See #4205.
-	runTests(c, nil, func(dbt *DBTest) {
+	runTests(c, func(config *mysql.Config) {
+		config.Strict = false
```

Comment:
why change this?

Reply:
if strict is ture, dbt.mustExec("drop table if exists test;") will fail.Because strict settings, warnings are treated as erroneous returns.
```
vendor/github.com/go-sql-driver/mysql/packets.go
func (mc *mysqlConn) handleOkPacket(data []byte) error {
....
// warning count [2 bytes]
if !mc.strict {
	return nil
}

pos := 1 + n + m + 2
if binary.LittleEndian.Uint16(data[pos:pos+2]) > 0 {
	return mc.getWarnings()
}
return nil	
}	
```

Reply:
Other test cases are similar settings.
```
server/server_test.go
func runTestLoadData(c *C, server *Server) {
...
	// support ClientLocalFiles capability
	runTestsOnNewDB(c, func(config *mysql.Config) {
		config.AllowAllFiles = true
		config.Strict = false
	}, "LoadData", func(dbt *DBTest) {
		dbt.mustExec("create table test (a varchar(255), b varchar(255) default 'default value', c int not null auto_increment, primary key(c))")
		rs, err1 := dbt.db.Exec("load data local infile '/tmp/load_data_test.csv' into table test")
		dbt.Assert(err1, IsNil)
		lastID, err1 := rs.LastInsertId()
...		
```